### PR TITLE
feat: new nabvar design

### PR DESCRIPTION
**Pull request by urielPalomares**
Captura de Pantalla 2022-12-02 a la(s) 12 55 08
Captura de Pantalla 2022-12-02 a la(s) 12 55 18

### CODE REVIEW

File: components/layout/navbar.module.css [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
-  justify-content: flex-end;
-  margin: 5px 10px;
+  padding: 1rem;
+  justify-content: space-between;
 }
 
 .avatar {
   box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
   cursor: pointer;
+  background: #1c6759 !important;
```
Comment (Andres2D) [inline components/layout/navbar.module.css:10]: Bad practice, use the chakra ui property.

File: components/layout/navbar.module.css [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+  align-items: center;
+  color: #333;
+}
+
+.avatarMenu {
+  margin-top: 2rem;
+  box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
+  cursor: pointer;
+  min-width: 5rem !important;
+  height: 5rem !important;
```
Comment (Andres2D) [inline components/layout/navbar.module.css:25]: Same here

File: components/layout/navbar.module.css [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+}
+
+.username {
+  padding-top: 10% ;
```
Comment (Andres2D) [inline components/layout/navbar.module.css:29]: This could be a static value.

File: components/layout/navbar.module.css [hunk 4]
```diff
@@ -1,19 +1,40 @@
...
+  /* padding-bottom: 5% ; */
```
Comment (Andres2D) [inline components/layout/navbar.module.css:30]: Remove this comment.

File: components/layout/navbar.tsx [hunk 5]
```diff
@@ -32,15 +32,23 @@ const Navbar: NextPage = () => {
 
   return (
     <>
-      <div className={styles.container}>
+    <div className={styles.container}>
+        <Avatar 
+          className={styles.avatar}
+          src='/images/cotejoAppUI.png'
```
Comment (Andres2D) [inline components/layout/navbar.tsx:38]: Try to save the file with the same standar than the others.
```suggestion
          src='/images/cotejo-app-ui.png'
```

**Review by Andres2D — CHANGES_REQUESTED** [review]

File: components/layout/navbar.module.css (hunk 1 above)
Comment (urielPalomares) [reply components/layout/navbar.module.css]: Listo

File: components/layout/navbar.module.css (hunk 2 above)
Comment (urielPalomares) [reply components/layout/navbar.module.css]: Ready

File: components/layout/navbar.module.css (hunk 3 above)
Comment (urielPalomares) [reply components/layout/navbar.module.css]: Ready

File: components/layout/navbar.module.css (hunk 4 above)
Comment (urielPalomares) [reply components/layout/navbar.module.css]: Ready

File: components/layout/navbar.tsx (hunk 5 above)
Comment (urielPalomares) [reply components/layout/navbar.tsx]: Ready

**Review by Andres2D — APPROVED** [review]

**Pull request merged**